### PR TITLE
Json

### PR DESCRIPTION
**Pull request by kathytbui**
This PR closes 

This PR fixes the way the endpoints are rendered on a response so that they follow JAON:API specification
Following the following format:
```json
{
    data: {
        type:
        id:
        attributes: {
        }
    }
{
```

This also takes care of the supplies and scheduled data inputs to the database. 

Error handling is also applied where it can be.

###What we are still missing:
- No testing
- Travis is still failing

### CODE REVIEW

File: api/database/models.py [hunk 1]
```diff
@@ -165,3 +173,50 @@ def update(self):
...
+        type_ = 'reminders'
+        strict = True
+    id = fields.Integer(dump_only=True)
+    user_id = fields.Integer(required=True)
+    title = fields.String(required=True)
+    supplies = fields.String(required=True)
+    show_supplies = fields.Boolean(required=True)
+    creation_date = fields.DateTime()
+    location_reminder = ma.Nested(LocationsSchema)
+    scheduled_reminder = ma.Nested(SchedulesSchema)
```
Comment (jordy1611) [inline api/database/models.py:222]: New schema looks good!

File: api/resources/reminders.py [hunk 2]
```diff
@@ -143,32 +118,45 @@ def patch(self, *args, **kwargs):
...
+            reminder = Reminder.query.filter_by(id=json_data['id']).first()
         except NoResultFound:
             return abort(404)
 
         reminder.delete()
-        return {}, 204
+        return {'message': 'Reminder successfully deleted'}, 200
+
+class ReminderResource(Resource):
+    def get(self, *args, **kwargs):
```
Comment (jordy1611) [inline api/resources/reminders.py:144]: Adding the *args and *kwargs looks like it lets the FE send whatever they want with less restrictions

**Review by jordy1611 — APPROVED** [review]
Looks great!

File: api/database/models.py (hunk 1 above)
Comment (jKwibe) [reply api/database/models.py:222]: Nice @Kathybui732

**Pull request merged**